core/src/test/scala/org/apache/spark/storage/BlockManagerSuite.scala

```diff
@@ -1932,22 +1932,29 @@ class BlockManagerSuite extends SparkFunSuite with Matchers with BeforeAndAfterE
     assert(master.getLocations(blockIdLarge) === Seq(store1.blockManagerId))
   }
 
-  test("test migration of shuffle blocks during decommissioning") {
-    val shuffleManager1 = makeSortShuffleManager()
+  private def testShuffleBlockDecommissioning(maxShuffle: Option[Int], migrate: Boolean) = {
+    maxShuffle.foreach{ b =>
+      conf.set(STORAGE_REMOTE_SHUFFLE_MAX_DISK.key, s"${b}b")
```

Comment:
Probably rename `maxShuffle` to `maxShuffleDiskSize` and rename `b` to `diskSize`

Reply:
Sure